addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -507,6 +509,58 @@ options.registry.navTabs = options.Class.extend({
     },
 });
 
+options.registry.NavTabsStyle = options.Class.extend({
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Set the style of the tabs.
+     *
+     * @see this.selectClass for parameters
+     */
+    setStyle: function (previewMode, widgetValue, params) {
+        const $nav = this.$target.find('ul.nav');
+        const isPills = widgetValue === 'pills';
+        $nav.toggleClass('nav-tabs card-header-tabs', !isPills);
+        $nav.toggleClass('nav-pills', isPills);
+        this.$target.find('.s_tabs_nav').toggleClass('card-header', !isPills).toggleClass('mb-3', isPills);
+        this.$target.toggleClass('card', !isPills);
+        this.$target.find('.s_tabs_content').toggleClass('card-body', !isPills);
+    },
+    /**
+     * Horizontal/vertical nav.
+     *
+     * @see this.selectClass for parameters
+     */
+    setDirection: function (previewMode, widgetValue, params) {
+        const isVertical = widgetValue === 'vertical';
+        this.$target.find('.s_tabs_main').toggleClass('row s_col_no_resize s_col_no_bgcolor', isVertical);
+        this.$target.find('ul.nav').toggleClass('flex-column', isVertical);
+        this.$target.find('ul.nav > li > a ').toggleClass('py-2', isVertical);
```

Comment:
Remove extra space in selector. But in fact, should not it be `.nav-link` or something like ?